apps/stripe/src/modules/stripe/stripe-publishable-key.ts

```diff
@@ -13,6 +14,10 @@ export class StripePublishableKey {
     return new StripePublishableKey(args.publishableKey);
   }
 
+  static createFromAppConfig(args: { appConfig: AppConfig }) {
+    return new StripePublishableKey(args.appConfig.getStripePublishableKey());
+  }
```

Comment:
Hmm, it's a little distributed that every value has it's own constructor and "root" config is passed

Don;t you think this is better?

```
new AppConfig().getPublishableKey()

```